packages/sidecar/src/cli/run.ts

```diff
@@ -16,6 +17,48 @@ import tail, { type OnItemCallback } from "./tail.js";
 
 const SPOTLIGHT_VERSION = process.env.npm_package_version || "unknown";
 
+/**
+ * Detect if there's a package.json with runnable scripts
+ */
+function detectPackageJson(): { scriptName: string; scriptCommand: string } | null {
+  try {
+    const scripts = JSON.parse(readFileSync("./package.json", "utf-8")).scripts;
+    const scriptName = scripts.dev ?? scripts.develop ?? scripts.serve ?? scripts.start;
+    if (scriptName) {
+      const detectedScript = ["dev", "develop", "serve", "start"].find(name => scripts[name]);
+      return { scriptName: detectedScript || "dev", scriptCommand: scriptName };
+    }
+  } catch {
+    // pass
+  }
+  return null;
+}
+
+/**
+ * Prompt the user to choose between Docker Compose and package.json
+ */
+async function promptUserChoice(): Promise<"docker" | "package"> {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+  return new Promise(resolve => {
+    console.log("\n⚠️  Both Docker Compose and package.json detected!");
+    console.log("\nWhich would you like to use?");
+    console.log("  1) Docker compose");
+    console.log("  2) package.json");
+
```

Comment:
not using the logger so we don't get the `🔎 [Spotlight]` prefix for these lines.